tests/unit/src/main/scala/tests/BaseDapSuite.scala

```diff
@@ -8,8 +8,48 @@ import scala.concurrent.Future
 import scala.meta.internal.metals.debug.DebugWorkspaceLayout
 import scala.meta.internal.metals.debug.Stoppage
 import scala.meta.internal.metals.debug.TestDebugger
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.GlobalTrace
+import scala.meta.internal.metals.debug.DebugProtocol
+import scala.util.Failure
+import scala.util.Success
 
 abstract class BaseDapSuite(suiteName: String) extends BaseLspSuite(suiteName) {
+
+  private val dapClient =
+    GlobalTrace.protocolTracePath(DebugProtocol.clientName)
+  private val dapServer =
+    GlobalTrace.protocolTracePath(DebugProtocol.serverName)
+
+  override def beforeAll(): Unit = {
```

Comment:
The traces should be per test-case, not test-suite (note: I an guessing the correct name of the method ;) )
```suggestion
  override def beforeEach(): Unit = {
```

Reply:
Changed